runtime/parachains/src/builder.rs

```diff
@@ -535,8 +535,8 @@ impl<T: paras_inherent::Config> BenchBuilder<T> {
 						validation_code_hash,
 					},
 					commitments: CandidateCommitments::<u32> {
-						upward_messages: Vec::new(),
-						horizontal_messages: Vec::new(),
+						upward_messages: vec![].try_into().unwrap(),
```

Comment:
Any particular reason why you're using `try_into().unwrap()` here instead of just `Default::default()` (which would eliminate the `unwrap`)?